apps/fabric-website/src/components/IconGrid/IconGrid.tsx

```diff
@@ -32,7 +32,7 @@ export class IconGrid extends React.Component<IIconGridProps, IIconGridState> {
 
     return (
       <div>
```

Comment:
Can you please add a change file?

I have removed the hardcoded string and fixed the build error for you.